src/dials/util/export_shelx.py

```diff
@@ -156,13 +156,13 @@ def _write_ins(experiment_list, best_unit_cell, ins_file):
             f"TITL {sg.type().number()} in {sg.type().lookup_symbol().replace(' ','')}\n"
         )
         f.write(
-            "CELL {:8.5f} {:8.4f} {:8.4f} {:8.4f} {:8.3f} {:8.3f} {:8.3f}\n".format(
+            "CELL {:7.5f} {:9.5f} {:9.5f} {:9.5f} {:8.4f} {:8.4f} {:8.4f}\n".format(
```

Comment:
I wonder if we should check the number of digits needed to express the unit cell and derive the format accordingly?

i.e. just saying if it _does_ have a crazy unit cell like 10, 10, 1001, 90, 90, 90 we could probably drop the precision by 1dp.

Reply:
Probably the correct behaviour is to check the esds to determine the precision required. I don't believe any of the SHELXx programs *require* fixed-width here?

Reply:
From what I can find online it seems that `.ins` files are read in free format by most (all?) shelx programs

Reply:
Personally I think matching the XPREP output would be fine rather than adding extra complexity